src/OrchardCore.Modules/OrchardCore.Security/Options/ContentSecurityPolicyOriginValue.cs

```diff
@@ -0,0 +1,11 @@
+namespace OrchardCore.Security.Options
+{
+    public class ContentSecurityPolicyOriginValue
+    {
+        public const string Any = "*";
+
+        public const string None = "'none'";
+
+        public const string Self = "'self'";
```

Comment:
Do we need another value to say that we don't want the related headers at all.

Or, I don't think so but maybe the opposite, even if we didn't do a first settings save, so without any values, maybe we should assume they are equal to None and still populate all headers with `=()`, which is not the case.

Reply:
> Do we need another value to say that we don't want the related headers at all.

I don't think so, probably stripping them from being adding is better